```


```
# Eight forms of the Amati relation that are most often published.

$
\begin{align}
(1) \ \ \ \ \ \log_{10}(E_{iso}) = \xi + \gamma \log_{10}(E^*_{pk})
\end{align}
$

$
\begin{align}
(2) \ \ \ \ \ E_{iso} = 10^{\xi} \ E^{* \ \gamma}_{pk}  
\end{align}
$

$
\begin{align}
(3) \ \ \ \ \ \log_{10}\left(\frac{E_{iso}}{N}\right) = \xi_N + \gamma \log_{10}(E^*_{pk})
\end{align}
$

$
\begin{align}
(4) \ \ \ \ \ \frac{E_{iso}}{N} = 10^{\xi_N} \ E^{* \ \gamma}_{pk}  
\end{align}
$

$
\begin{align}
(5) \ \ \ \ \ \log_{10}(E^*_{pk}) = -\frac{\xi}{\gamma} + \gamma^{-1} \log_{10}(E_{iso})
\end{align}
$

$
\begin{align}
(6) \ \ \ \ \ E^*_{pk} = 10^{-\frac{\xi}{\gamma}} \ E_{iso}^{\gamma^{-1}} 
\end{align}
$


$
\begin{align}
(7) \ \ \ \ \ \log_{10}(E^*_{pk}) = -\frac{\xi_N}{\gamma} + \gamma^{-1} \log_{10}\left(\frac{E_{iso}}{N}\right)
\end{align}
$

$
\begin{align}
(8) \ \ \ \ \ E^*_{pk} = 10^{-\frac{\xi_N}{\gamma}} \ \left( \frac{E_{iso}}{N}\right)^{\gamma^{-1}} 
\end{align}
$


$
\begin{align}
\xi = \xi_N + \log_{10}(N)
\end{align}
$


$
\begin{align}
\xi_N = \xi - \log_{10}(N)
\end{align}
$

## Important note(s):
- In all equations, $E^*_{pk} = E_{pk}(1+z)$. If $E^*_{pk}$ is raised to a power, then it applies to both $E_{pk}$ and $(1+z)$ individually. 
I.e., $E^{* \ \gamma}_{pk} = E^{\gamma}_{pk}(1+z)^{\gamma}$




## Notes to explain the differences.
- We provide linearized forms and power forms. Equation (1) is a linearized form and equation (2) is a power form of the same equation. A shortcut to go between the two is to use:
$\log(y) = c \log(x) + \log(a) \ \ \  \Leftrightarrow \ \ \ y=aX^c$

- We provide forms for when both Epeak and Eiso are the dependent variables (i.e., left side of = sign) and also the y-axis variable on a plot. 
- We provide forms where Eiso is normalized (or scalled) to a certain energy value and when it is not. Equation (1) is a non-scalled form and equation (3) is its scalled version. This is why you will see $\xi_N$ instead of $\xi$ (they are NOT the same value but can be derived from one another). More on this in a moment. You can see $\frac{E_{iso}}{N}$ in the scalled equations, and $N$ should be the energy the equation is normalized to. For example, $N$ would be $1E52$ if the equation was normalized to $E_{iso} \sim 1\times10^{52}$ erg. However, most authors neglect this in their equation. For example:

$
\begin{align}
\log_{10}(E^*_{pk}) = 2 + 0.5 \log_{10}(E_{iso})
\end{align}
$

should more appropriately be written as:

$
\begin{align}
\log_{10}\left(\frac{E^*_{pk}}{\text{keV}}\right) = 2 + 0.5 \log_{10}\left(\frac{E_{iso}}{1\times10^{52}\text{erg}}\right)
\end{align}
$

where that $N$ variable is the $1\times10^{52}$ erg. 




### Notes on $\xi$ verses $\xi_N$

$\xi_N$ depends only on the $E_{iso}$ normalization and the value of $\xi$. 
More often than not, the equation will be scalled and you will have $\xi_N$ and need to derive $\xi$ from it.
Below are those conversion equations:

$
\begin{align}
\xi = \xi_N + \log_{10}(N)
\end{align}
$


$
\begin{align}
\xi_N = \xi - \log_{10}(N)
\end{align}
$

where $N$ is the energy that $E_{iso}$ is normalized to. 


#### Converting between Linearized and Power forms of the equation. 
These are the steps that we skipped by using the shortcut, see above.


Start with the power version:  $y = aX^c$
1. Log both sides. 
 $\log(y) = \log(aX^c)$
2. Use rule #1 from https://mathbench.umd.edu/modules/misc_scaling/page11.htm 
$\log(y) = \log(aX^c) = \log(a) + \log(X^c)$
3. Use rule #2 from https://mathbench.umd.edu/modules/misc_scaling/page11.htm 
$\log(y) = \log(a) + \log(X^c) = \log(a) + c\log(X) $
4. Rearrange and compare this to the equation of a line.
$\log(y) = c\log(X) + \log(a) \ \ $ or $ \ \ y=mx+b$. 

Where $c$ would be the slope and $\log(a)$ the y-intercept. 

In [70]:
import numpy as np


def get_xi_N(xi, N):
    return xi - np.log10(N)


def get_xi(xi_N, N):
    """
    When your function is normalized to some Eiso energy, you can 
    find out what xi should be based on the xi_N given in the 
    Amati equation and the normalization energy. 
    """
    return xi_N + np.log10(N)

```




```
# Examples on deciding which form you have based on your equation.

```


```
$
\begin{align}
\log_{10}(E_{iso}) = 47.93 + 2.04 \log_{10}(E^*_{pk}) 
\end{align}
$

#### This is equation (1) above and you have $\xi=47.93$. This equation is not scalled. 

```


```
$
\begin{align}
\log_{10}(E_{iso}) = -4.07 + 2.04 \log_{10}(E^*_{pk}) 
\end{align}
$

#### This is equation (3) above and you have $\xi_N=-4.07$. This equation is scalled. 

In [28]:
get_xi(xi_N=-4.07, N=1E52)

47.93

```


```
$
\begin{align}
\log_{10}(E^*_{pk}) = -26.62 + 0.551 \log_{10}(E_{iso}) 
\end{align}
$

#### This is equation (5) above and you have $-\frac{\xi}{\gamma}=-26.62$. This equation is NOT scalled. 
To find the scalled version of it, use $\gamma^{-1}=0.551$ and $-\frac{\xi}{\gamma}=-26.62$ to get $\gamma=(1/0.551)$ and $\xi=48.312159709618875$. Solve for $\xi_N$, which is $\xi_N=-3.687840290381125$, which gives $-\frac{\xi_N}{\gamma}=2.032$.

In [50]:
26.62/(0.551)

48.312159709618875

In [51]:
get_xi_N(xi=48.312159709618875, N=1E52) #, 0.09393346379648193/(1./.551)

-3.687840290381125

In [52]:
-(-3.687840290381125/(1./0.551))

2.032

```


```
$
\begin{align}
\log_{10}(E^*_{pk}) = 2.032 + 0.551 \log_{10}(E_{iso}) 
\end{align}
$

#### This is equation (7) above and we actually derived this version from the last example. This equation is the scalled version of the last. 
To find the unscalled version of it (make sure our work is consistent), use $\gamma^{-1}=0.551$ and $-\frac{\xi_N}{\gamma}=2.032$ to get $\gamma=(1/0.551)$ and $\xi_N=-3.687840290381125$. Solve for $\xi$, which is $\xi=48.312159709618875$, which gives $-\frac{\xi}{\gamma}=-26.62$.

In [53]:
-(2.032*(1./0.551))

-3.687840290381125

In [54]:
get_xi(xi_N=-3.687840290381125, N=1E52)

48.312159709618875

In [55]:
-(48.312159709618875/(1./0.551))

-26.62

```


```
$
\begin{align}
\log_{10}(E^*_{pk}) = 2 + 0.5 \log_{10}(E_{iso}) 
\end{align}
$

$
\begin{align}
\log_{10}(E^*_{pk}) = 1.933 + 0.555 \log_{10}(E_{iso}) 
\end{align}
$

#### These are both examples of equation (7) above, which is the most popular form published for Amati's relation. The top one here is what Amati et al. (2013) considers their relation in the text. The bottom one is our version of the equation for when we fit a power-law to the 34 LGRBs in our data sample. 

## By now it should be clear how we get from a non-scalled equation to a scalled version of it. And vice versa. 

```


```

# Solve the Amati Equation

In [75]:
def solve_amati_for_eiso(epeakRest, xi, gamma, N=None):
    """
    The value of xi_N depends on the normalization of Eiso.
    We calculate it within the function and have it printed to 
    the screen when a normalization is used. 
    
    Most often, if you have a normalization, you also have 
    the proper value of xi_N. If you have it, use it in the place 
    of xi. 
    """
    if N is not None:
        xi_N = get_xi_N(xi, N)
        print(xi_N)
        return ((10**xi_N) * (epeakRest**gamma)) * N
    else:
        return (10**xi) * (epeakRest**gamma)
    
def solve_amati_for_epeakRest(eiso, xi, gamma, N=None):
    """
    The value of xi_N depends on the normalization of Eiso.
    We calculate it within the function and have it printed to 
    the screen when a normalization is used. 
    
    Most often, if you have a normalization, you also have 
    the proper value of xi_N. If you have it, use it in the place 
    of xi. 
    """
    if N is not None:
        xi_N = get_xi_N(xi, N)
        #print(xi_N)
        # Use xi_N in place of xi
        return (10**(-xi_N/gamma)) * ((eiso/N)**(gamma**-1))
    else:
        return (10**(-xi/gamma)) * (eiso**(gamma**-1))


In [76]:
solve_amati_for_epeakRest(eiso=3.E54, xi=47.93, gamma=(1./0.49))

1614.6909636021576

In [77]:
solve_amati_for_epeakRest(eiso=3.E54, xi=47.93, gamma=(1./0.49), N=1E51)

1614.690963602152

In [65]:
solve_amati_for_eiso(epeakRest=346.84244786843345, xi=47.93, gamma=(1./0.49))

1.3000000000000102e+53

In [66]:
solve_amati_for_eiso(epeakRest=346.84244786843345, xi=47.93, gamma=(1./0.49), N=1E51)

-3.0700000000000003


1.3000000000000102e+53

# Most of the time we know $\xi_N$, since the equation is presented in that form. 

\begin{equation}
\log_{10}(E^*_{pk}) = -\frac{\xi_N}{\gamma} + \gamma^{-1} \log_{10}\left(\frac{E_{iso}}{N}\right)
\end{equation}

\begin{equation}
\log_{10}(E^*_{pk}) = 1.933 + 0.555 \log_{10}(E_{iso})
\end{equation}

$-\frac{\xi_N}{\gamma} = 1.933$

$\gamma^{-1}=0.555$

thus,

$\gamma= \frac{1}{0.555}$

$-\frac{\xi_N}{\frac{1}{0.555}} = -\xi_N \cdot 0.555 = 1.933 \ \ \ $
thus,
$ \ \ \ \xi_N=-\left(1.933 \cdot \frac{1}{0.555}\right) = -3.4828828828828824$

To get $\xi$ we use $\xi = \xi_N + \log_{10}(N)$ where $N=1\times10^{52}$ erg. 

$\xi=48.51711711711712$


In [67]:
-(1.933*(1.0/0.555))

-3.4828828828828824

In [68]:
-(1.933*(1.0/0.555)) + np.log10(1E52)

48.51711711711712

In [69]:
get_xi(xi_N=-3.4828828828828824, N=1E52)

48.51711711711712

\begin{equation}
\log_{10}(E^*_{pk}) = 48.517 + 0.555 \log_{10}(E_{iso})
\end{equation}